model_inference/bird_predictor.py

```diff
@@ -17,12 +17,18 @@
         6: 'Цыпленок'
     }
 
+def download_file_from_url(url, destination_path):
+    response = requests.get(url, stream=True)
+    
+    with open(destination_path, 'wb') as file:
+        for chunk in response.iter_content(chunk_size=128):
+            file.write(chunk)
+
 def model_upload():
   output = 'bird_recognition.h5'
```

Comment:
Why is this thing called "output"? If I understand the situation correctly, it is the model weights. Can you rename this variable according to its role? In this situation, I would first think it is the prediction output from the model.